FAQ.md

```diff
@@ -476,6 +476,17 @@ O formalnościach związanych z zaliczaniem praktyk więcej [tutaj](https://foru
 
 </details>
 
+<summary>Czy pracę dyplomową można pisać zespołowo?</summary>
+
+> Tak, praca dyplomowa może być realizowana zespołowo. Prace licencjackie oraz inżynierskie można realizować w zespołach dwu- i trzyosobowych. Prace magisterskie można realizować w zespołach dwuosobowych.
+>
+> Jeżeli praca dyplomowa jest realizowana w zespole, powinna być odpowiednio większa od pracy stworzonej przez jedną osobę.
+> 
+> UWAGA: praca licencjacka/inżynierska zrealizowana zespołowo nie zalicza projektu zespołowego.
```

Comment:
To można dodatkowo dopisać jako osobny punkt, przy sekcji o projekcie zespołowym

Reply:
O, z jakiegoś powodu myślałem, że tego nie mamy. Tak, zdecydowanie lepiej coś powtórzyć parę razy niż żeby miało gdzieś zniknąć :)